server/utils/validation/applicantValidation.js

```diff
@@ -0,0 +1,34 @@
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  address: yup.string().trim(),
+  cohorts: yup.string().trim(),
+  mobileNumber: yup.string().trim(),
+  age: yup.number().integer().min(18),
+  motivation: yup.string().trim(),
+  englishSpeaking: yup.string().trim(),
+  englishUnderstanding: yup.string().trim(),
+  employmentStatus: yup.string().trim(),
+  jobTitle: yup.string().trim(),
+  university: yup.string().trim(),
+  specialization: yup.string().trim(),
+  cvLink: yup.string().trim(),
+  codingExperience: yup.string().trim(),
+  primaryMotivation: yup.string().trim(),
+  caReading: yup.string().trim(),
+  available: yup.boolean(),
+  githubLink: yup.string().trim(),
+  joinDiscord: yup.boolean(),
+  freeCodeCampPoints: yup.number().integer().min(0).max(1500),
+  freeCodeCampTopics: yup.boolean(),
+  freeCodeCampLink: yup.string().trim(),
+  codeWarsLink: yup.string().trim(),
+  codeWarsKyu: yup.string().trim(),
+  technicalTasks: yup.boolean(),
+  technicalTasksLinks: yup.string().trim(),
+  projectGithubLink: yup.string().trim(),
```

Comment:
Use url validation instead of string from yup to validate the link (freecodecamp, codewars and github)

https://github.com/jquense/yup/blob/master/README.md#stringurlmessage-string--function-schema